hack/update-bindata.sh

```diff
@@ -28,25 +28,14 @@ if [[ ! -d "${KUBE_ROOT}/examples" ]]; then
 	exit 1
 fi
 
-# Setup bindata if not already in the system.
-# For separation of concerns, download first, then install later.
-if ! git config -l | grep -q "user.name" && ! git config -l | grep -q "user.email" ; then
-    git config --global user.name bindata-mockuser
-    git config --global user.email bindata-mockuser@example.com
-fi
-
-go get -u github.com/jteeuwen/go-bindata/... || echo "go-bindata get failed, possibly already exists, proceeding"
-go install github.com/jteeuwen/go-bindata/... || echo "go-bindata install may have failed, proceeding anyway..."
-
-if [[ ! -f ${GOPATH}/bin/go-bindata ]]; then
-	echo "missing bin/go-bindata"
-	echo "for debugging, printing search for bindata files out..."
-	find ${GOPATH} -name go-bindata
+if ! which go-bindata &>/dev/null ; then
```

Comment:
IMO `GOPATH` should be the only pre-requisite, and GOBATH/bin should just be used as necessary.  but thats not a huge deal either way.